_author/christophe_b.md

```diff
@@ -0,0 +1,5 @@
+---
+fullname: Christophe Bonraisin
+role: Intrapreneur à Pôle emploi
+---
+Irrité par les irritants, pugnace pour chercher à les résoudre, avec <i class="large smile icon"/>
```

Comment:
`<i>` n'est [pas auto-fermant](http://www.w3.org/TR/html5/text-level-semantics.html#the-i-element).